oracle/oracle-how-to/deploy-oke-nodes-using-ubuntu-images.rst

```diff
@@ -73,93 +73,196 @@ When registering images, the :guilabel:`Launch mode` is an option to configure.
       
-      Then, create a placement configuration file to specify where in Oracle Cloud the managed node pool should be created and save the file as ``placement-config.json``.
+.. code:: bash
+  
+  # Create internet gateway
+  oci network internet-gateway create \
+    --compartment-id <compartment-id> \
+    --vcn-id <vcn-id> \
+    --is-enabled true
+
+Next up, refer to the Oracle documentation for `security lists`_ for information on creating security rules for the nodes, control plane, and service load balancer. You can create a security list using the following command:
+
+.. code:: bash
+  
+  oci network security-list create \
+  --compartment-id <compartment-id> \
+  --vcn-id <vcn-id> \
+  --display-name <security-list-name> \
+  --egress-security-rules <rules> \
+  --ingress-security-rules <rules>
+                
+Now that you have the VCN, gateways, and security lists, you can create the route table and the subnets. Typically, you will need a nodes subnet, a control plane subnet, and a service load balancer subnet.
+
+You can create a route table and a subnet using the following commands.
       
-      .. code:: json 
+.. code:: bash
+  
+  # Create public route
+  oci network route-table create \
+      --compartment-id <compartment-id>\
+      --vcn-id <vcn-id> \
+      --display-name <route-table-name> \
+      --route-rules <route-rules-with-internet-gateway>
+
+  # Create nodes subnet                
+  oci network subnet create \
+      --compartment-id <compartment-id>\
+      --vcn-id <vcn-id> \
+      --display-name <nodes-subnet-name> \
+      --cidr-block <subnet-cidr-block> \
+      --route-table-id <route-table-ocid> \
+      --security-list-ids <nodes-seclist-ocid>
+      ...
       
-         [{
-           "compartmentId":"<compartment-id>",
-           "availabilityDomain":"<availability-domain>",
-           "subnetId":"<subnet-id>"
-         }]
+  # Similarly, create a control plane subnet
+  # and a service load balancer subnet
+
+Create the OKE cluster
+~~~~~~~~~~~~~~~~~~~~~~~
+
+To create the OKE cluster, you will need to provide the compartment ID, the VCN OCID, and the subnets for the control plane and service load balancer. For more details on cluster creation, please refer to the Oracle documentation on `creating a cluster`_.
+
+The following command will create the OKE cluster.
+
+.. code:: bash
+  
+  oci ce cluster create \
+      --compartment-id <compartment-id> \
+      --name <cluster-name> \
+      --kubernetes-version <kubernetes-version> \
+      --vcn-id <vcn-ocid> \
+      --cluster-pod-network-options <cluster-network-options> \
+      --endpoint-subnet-id <control-plane-subnet-ocid> \
+      --service-lb-subnet-ids "[<service-lb-subnet-ocid>]"
+      ...
       
+Once the cluster is created, you can create a kubeconfig file to access the cluster through `kubectl`. The following command will generate the kubeconfig file:
```

Comment:
@Gisaldjo did you intend to have kubectl italics like \`kubectl\` or as a code block like \`\`kubectl\`\`